plugins/modules/rpm_ostree_pkg.py

```diff
@@ -104,6 +118,12 @@
     returned: always
     type: str
     sample: 'rpm-ostree uninstall --allow-inactive --idempotent --unchanged-exit-77 nfs-utils'
+needs_reboot:
+    description: Determine if machine needs a reboot to apply current changes.
+    returned: always
```

Comment:
```suggestion
    returned: success
```
This won't get added to the output if the module fails :)

Reply:
My mistake, fixing now